### ISSUE TITLE
Serve one site (with one langague) to two or more domains.

### ISSUE THREAD

**Issue by Aquive** [statamic/ideas#1012]
As the title suggests. A good example would be to have a site with German content to be served at:

- site.de
- site.at

**Comment by duncanmcclean** [comment]
What's your use case for this? Would it be serving exactly the same content?

**Comment by AndreasSchantl** [comment]
> What's your use case for this? Would it be serving exactly the same content or just the same language?

I guess both. I completely agree. There are regions like Germany, Austria and Switzerland that do speak the same language. There's no need for changing content.

@Aquive have you checked out the Auto Propogate functionality? This should serve your needs as well as a workaround.

**Comment by Aquive** [comment]
Yes indeed just serving the same content to different TLD's. I will check the workaround. Thanks @AndreasSchantl

**Comment by wm-simon** [comment]
Other cms have an option like `aliases`, where your can add multiple domains. Perhaps this could be integrated to the `sites.php`.

```
        'default' => [
            'name' => config('app.name'),
            'locale' => 'de_DE',
            'url' => 'domain1.tld',
            'aliases' => 'domain2.tld',
        ],
```

**Comment by Aquive** [comment]
I am out of office until the 7th of August. The fastest way to get answer for support issues is by emailing to ***@***.***

For other issues, you may contact one of my colleagues, Jeroen de Reus ***@***.***) or Jan Zelig ***@***.***).
